src/l10n/helptexts.nb.js

```diff
@@ -331,7 +334,7 @@ Velg Feed dersom det er en distribusjon hvor endringer hentes gjennom f.eks. RSS
   Distribution_format: {
     abstract:
       'Velg format(er) fra IANAs liste over offisielle medietyper. Dersom formatet ikke finnes i listen kan du angi eget.',
-    description: `Skriv inn format og avlsutt med TAB eller Enter. Vi anbefaler at du bruker de samme formatbeskrivelsene som du finner i lenken nedenfor. (Obs: Du må selv kopiere over formatet fra listen.) 
```

Comment:
Bør legge til lenke for denne også:
https://data.norge.no/specification/dcat-ap-no/#Distribusjon-format

Reply:
@lillelars For "hasQualityAnnotation" og "accrualPeriodicity" tok jeg utgangspunkt i navnene til delene i hjelpetekstfilen, og syntes de passet når jeg leste beskrivelsen i spesifikasjonen, men jeg er ikke sikker på om det er riktig. 
Jeg har nå lagt til lenke for distribusjon.